application/views/template.php

```diff
@@ -48,16 +48,16 @@
 							$services_down = '';
 
 							// Check LMD.
-							exec('service lmd status', $output, $return_var_lmd);
+							exec('/usr/bin/systemctl status lmd.service', $output, $return_var_lmd);
 							// Check Livestatus.
-							exec('service naemon status', $output, $return_var_le);
+							exec('/usr/bin/systemctl status naemon.service', $output, $return_var_le);
 
-							if($return_var_lmd !== 0 && $return_var_le !== 0){
+							if($return_var_lmd === 127 && $return_var_le === 127){
```

Comment:
Why do we need 127? From my understanding this is an exec error (command not founc) and not the systemctl return value.  refer to systemctl exit codes are in 
https://www.freedesktop.org/software/systemd/man/systemctl.html#Exit%20status